projects/core/tokens/disable-custom-scrollbar.ts

```diff
@@ -0,0 +1,8 @@
+import {InjectionToken} from '@angular/core';
+
+export const TUI_DISABLE_CUSTOM_SCROLLBAR = new InjectionToken(
```

Comment:
What do you think about 

```ts

interface TuiScrollbarMode {
  enabled: boolean
}

export const TUI_SCROLLBAR_MODE = new InjectionToken<TuiScrollbarMode>(
'custom options for scrollbar,
    {
        factory: () => { enabled: true }, // in the future it will be possible to expand with other properties
    },
);
```

Reply:
I would have liked to implement a config to control all scrollbars, including local bars. But thought that it’s too complicated to toggle them. However, it seems that if we implement some extra css classes, it’s possible to achieve this

Anyway, make sense, I’ll change the token

Reply:
`But thought that it’s too complicated to toggle them`

You can create handy helper like this:
https://github.com/Tinkoff/taiga-ui/blob/main/projects/kit/components/rating/rating-options.ts#L24

Reply:
the problem is in how to manage css styles, I mean

Reply:
I'm not sure we need to control `tui-scrollbar`. Why would somebody use it when they don't need a custom scrollbar?

Reply:
So we don’t need to implement options or this feature at all?

Don’t we use ‘tui-scrollbar’ in some components? So we can’t control it

Reply:
Right, this makes sense. Let's make a token that is used in all scrollbars then. We can have an object there:
```ts
{
  readonly mode: 'native' | 'custom';
}
```
for now. Later we might add `hidden` and `auto-hide` and maybe `size` or something, so having an object is reasonable. The way I see it we can add factory to `TUI_SCROLL_REF` and use `inject(DOCUMENT).scrollingElement` as default value. Then in `tui-scroll-controls` component we can set necessary styles to injected `TUI_SCROLL_REF`. Looks like this would also simplify `scrollbars` getter inside `TuiScrollControlsComponent`.

Reply:
There are some issues with `inject(DOCUMENT).scrollingElement`. It might return `null` in some cases: 

> When in quirks mode, the scrollingElement attribute returns the HTML body element if it exists and is potentially scrollable , otherwise it returns null. 

Should we handle this in factory or just return as it is?

The second problem is that `scrollingElement` is `Element`, should we cast it to `HTMLElement` in factory or in components?